viewer/packages/utilities/src/three/isBox3OnPositiveSideOfPlane.ts

```diff
@@ -0,0 +1,18 @@
+/*!
+ * Copyright 2021 Cognite AS
+ */
+import * as THREE from 'three';
+import { visitBox3CornerPoints } from './visitBox3CornerPoints';
+
+/**
+ * Returns true if all corner point of the box provides is on the "positive side"
+ * of the plane provided (i.e. the corner point has a non-negative signed distance to the
+ * plane)
+ */
+export function isBox3OnPositiveSideOfPlane(box: THREE.Box3, plane: THREE.Plane): boolean {
+  let planeAccepts = false;
+  visitBox3CornerPoints(box, boundCorner => {
+    planeAccepts = plane.distanceToPoint(boundCorner) >= 0 || planeAccepts;
```

Comment:
This is not the most efficient way to do this. It's fine I guess if you don't call this often. 

Better is to take the dot product betweem the normal of the plane and the vector from point on plane to the bound corner. If this is greater than 0, then the point is on the side of the plane which the normal is facing.